app/src/main/res/menu/rerun.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+	xmlns:app="http://schemas.android.com/apk/res-auto">
+<item
+		android:id="@+id/reRun"
+		android:icon="@drawable/rerun"
+		android:title="@string/TestResults_Overview_Tab_Label"
+		app:showAsAction="ifRoom|withText" />
+	<!-- Modal.ReRun.Websites.Run -->
```

Comment:
What is the purpose of this comment here?

Reply:
Updated